corehq/apps/toggle_ui/views.py

```diff
@@ -31,9 +32,28 @@ def page_url(self):
 
     @property
     def page_context(self):
+        toggles = list(all_toggles())
+        domain_counts = {}
+        user_counts = {}
+        for t in toggles:
+            counter = Counter()
+            try:
+                usage = Toggle.get(t.slug)
+            except ResourceNotFound:
+                domain_counts[t.slug] = 0
+                user_counts[t.slug] = 0
+                continue
```

Comment:
nitpick, but using a `try`/`else` construct would avoid needing a continue and make the control flow a little clearer